etc/mpv.profile

```diff
@@ -16,11 +16,15 @@ include /etc/firejail/disable-passwdmgr.inc
 
 caps.drop all
 netfilter
+nogroups
 nonewprivs
 noroot
 protocol unix,inet,inet6
 seccomp
+tracelog
 
 # to test
+ipc-namespace
```

Comment:
Those breakages were in mpv? I had issues with many other apps while using ipc-namespace but not in mpv